### DISCUSSION TITLE
How to add a /ads.txt

### DISCUSSION THREAD

**Discussion by willin** [misskey-dev/misskey#10739] [Q&A]
<img width="772" alt="image" src="https://user-images.githubusercontent.com/1890238/235494725-3fd4626e-a0ed-461c-ad84-e2a44d17981b.png">

`~/packages/backend/assets` not work.

besides, how to inject adsense sccripts

**Comment by mappi-pr** [comment]
@willin 
This will allow you to use it.

/packages/backend/src/server/web/ClientServerService.ts
<img width="478" alt="image" src="https://github.com/misskey-dev/misskey/assets/75538777/e8f7ead5-fa7f-4d75-be51-d3d5c0daece2">

```
fastify.get('/ads.txt', async (request, reply) => {
	return await reply.sendFile('/ads.txt', staticAssets);
});
```

■After fix
<img width="407" alt="image" src="https://github.com/misskey-dev/misskey/assets/75538777/b3b7d0e0-eb15-4360-996a-6ef2e1d8f20d">
